src/exam/messages.js

```diff
@@ -9,6 +9,10 @@ const messages = defineMessages({
     id: 'exam.apiError.supportText.withoutLink',
     defaultMessage: 'If the issue persists, please reach out to support for assistance, and return to the exam once you receive further instructions.',
   },
+  proctoredExamAccessDenied: {
+    id: 'exam.proctoredExamDenied',
+    defaultMessage: 'Cannot access proctored exams if user not eligible to receive certificates.',
```

Comment:
I think this can be more user-friendly - `You do not have access to proctored exams with your current enrollment`, or something like that?

Reply:
Sounds good. The message updated according to your suggestion.

Reply:
The new words screenshot
![dontHaveProctoredAccess](https://user-images.githubusercontent.com/16839373/154154926-463129d6-a467-46d8-b502-0bde70ac9e4e.jpeg)